tests/unit_tests/test_game_rating.py

```diff
@@ -984,10 +1047,292 @@ async def test_single_wrong_report_still_rated_correctly(game: Game, player_fact
     await game.on_game_end()
     await rating_service._join_rating_queue()
 
-    results = get_persisted_results(rating_service)
+    results, = get_persisted_results(rating_service)
     winning_ids = log_dict["teams"][str(log_dict["winning_team"])]
     for player_id, new_rating in results.ratings.items():
         if player_id in winning_ids:
-            assert new_rating.mu > old_rating
+            assert new_rating.mean > old_rating
         else:
-            assert new_rating.mu < old_rating
+            assert new_rating.mean < old_rating
+
+
+# NOTE: The following test_rating_adjustment_* tests were created by setting up
+# the inputs and copying the results to the expected assertions. They exist
+# primarily to document the current behaviour and detect bugs in case the
+# behaviour changes, however, there is no reason why the rating initialization
+# couldn't be changed and the values in these tests updated.
+async def do_test_rating_adjustment(
+    rating_service,
+    player_factory,
+    ratings,
+    expected_results
+):
+    team1 = set()
+    team2 = set()
+    teams = (team1, team2)
+    for player_id, ratings in ratings.items():
+        _ = player_factory(player_id=player_id)
```

Comment:
What does this line do?

Reply:
Nothing lol. It used to be used just for the player_id but then I refactored it.